src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -78,6 +78,8 @@ async def _run_formula(
             try:
                 msg = await formula.apply()
                 await sender.send(msg)
+            except asyncio.CancelledError:
+                break
             except Exception as err:  # pylint: disable=broad-except
                 logger.warning("Formula application failed: %s", err)
```

Comment:
Regarding: https://github.com/frequenz-floss/frequenz-sdk-python/pull/105#discussion_r1043148790

This should work:
```
try:  msg = await formula.apply()
except asyncio.CancelledError: (...)
except: except Exception as err: (...)
else:  await sender.send(msg)
```

Reply:
done

Reply:
This is ignoring errors while sending, but that's another topic (I still have some commits to add more exceptions to the channels repo but didn't have the time to clean it up for a PR :D).